packages/integrations/vercel/README.md

```diff
@@ -101,11 +102,56 @@ import vercel from '@astrojs/vercel/serverless';
 export default defineConfig({
   output: 'server',
   adapter: vercel({
-    analytics: true,
+    webAnalytics: {
+      enabled: true,
+    },
   }),
 });
 ```
 
+#### `beforeSend`
+
+Since functions can't be passed down via `astro.config.mjs`, you need to export the `beforeSend` function in a separate file inside your root called `vercel-web-analytics.ts`.
```

Comment:
I don't think we should tell users about our design decisions. We should just tell them "that's how you use this hook/function".

Reply:
Makes sense. Updated!